pcdsdevices/epics_motor.py

```diff
@@ -735,6 +737,27 @@ def flag_is_cleared(value=None, **kwargs):
             status_wait(st, timeout=timeout)
         return st
 
+    @property
+    def md(self):
+        if self._md is None:
+            raise AttributeError('Device does not have an attached md, '
+                                 'and was likely not initialized from happi')
+        return self._md
+
+    @md.setter
+    def md(self, new_md):
+        """ initialize attributes when md is set """
+        self._md = new_md
+        self._extra = self._md.extraneous
+        self._id = self._extra['_id']
+        self._stageidentity = self._extra['stageidentity']
+        self._pvbase = self._extra['pvbase']
+        if self._stageidentity == "NEW":
+            print(f"This is a new stage, parameter configuration does not exist yet. Please manually configure {self._id}.")
+        else:
+            IMS._setup_and_check_pmgr()
```

Comment:
To Zach's point here, we should probably wrap this in a `try: except:` to catch these exceptions.  Otherwise if we can't connect to the pmgr the loading process will get interrupted.  

In this function block, it seems like we should just return if an error is thrown from this point.  We got some metadata from happi but could not connect to pmgr, so we don't continue with setting the config (next line)